### PR TITLE
Adding install NApps dependecies

### LINKED ISSUE #60: Kytos napps install should take care of dependencies

### PR DESCRIPTION
**Pull request by macartur**
Fix #60

### CODE REVIEW

**Comment by beraldoleal** [comment]
@macartur we are doing a huge refactoring on napps, so I'm going to hold this for a while.

File: kytos/cli/commands/napps/api.py [hunk 1]
```diff
@@ -93,14 +93,27 @@ def uninstall(cls, args):
...
         mgr = NAppsManager()
-        for napp in args['<napp>']:
+        for napp in napps:
             mgr.set_napp(*napp)
             log.info('NApp %s:', mgr.napp_id)
             if not mgr.is_installed():
                 cls.install_napp(mgr)
             else:
                 log.info('  Installed.')
+            napp_dependencies = mgr.get_napp_dependentices()
```
Comment (cemsbr) [inline kytos/cli/commands/napps/api.py]: Should be `dependencies`.

File: kytos/utils/napps.py [hunk 2]
```diff
@@ -84,8 +84,35 @@ def get_disabled(self):
         enabled = set(self.get_enabled())
         return sorted(installed - enabled)
 
+    def get_napp_dependentices(self, user=None, napp=None):
```
Comment (cemsbr) [inline kytos/utils/napps.py]: `dependencies`

File: kytos/utils/napps.py [hunk 3]
```diff
@@ -84,8 +84,35 @@ def get_disabled(self):
...
+        """Generic method used to return a value from kytos.json.
+
+        Args:
+            user (string): A Username.
+            napp (string): A NApp name
+            key (string): Key used to get the value within kytos.json.
+        Returns:
+            meta (object): Value stored in kytos.json.
+        """
+        if key is None:
```
Comment (cemsbr) [inline kytos/utils/napps.py]: Shouldn't key be a required parameter?

File: kytos/utils/napps.py [hunk 4]
```diff
@@ -84,8 +84,33 @@ def get_disabled(self):
         enabled = set(self.get_enabled())
         return sorted(installed - enabled)
 
+    def dependentices(self, user=None, napp=None):
```
Comment (diraol) [inline kytos/utils/napps.py]: @macartur `s/dependentices/dependencies/` ou `s/dependentices/get_dependencies/`

**Pull request merged**